ERCS/erc-xyz.md

```diff
@@ -0,0 +1,166 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 7699
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.